gradle.properties

```
@@ -22,5 +22,5 @@ DEVELOPER_ID=heitorcolangelo
 DEVELOPER_NAME=Heitor Colangelo
 
 # VERSION
-VERSION_NAME=1.2.2
-VERSION_CODE=42
\ No newline at end of file
+VERSION_NAME=1.3.0
+VERSION_CODE=43
```

Add the following in order to prevent crashes in non-androidx projects:

```suggestion
android.enableJetifier=true
android.useAndroidX=true
```

As stated by Android developers:
> * `android.useAndroidX`: When set to true, the Android plugin uses the appropriate AndroidX library instead of a Support Library. The flag is false by default if it is not specified.
> * `android.enableJetifier`: When set to true, the Android plugin automatically migrates existing third-party libraries to use AndroidX by rewriting their binaries. The flag is false by default if it is not specified.